resources/templates/mustache/production/govuk-template.mustache.html

```diff
@@ -157,7 +157,15 @@
 
         <div id="skiplink-container">
             <div>
-                <a href="#skipto" class="skiplink">Skip to main content</a>
+                <a href="#skipto" class="skiplink">
+                    {{#isWelsh}}
+                    Ewch yn syth i‘r prif gynnwys
```

Comment:
Please fix the indentation of this line and change the quote to an encoded HTML entity `&rsquo;`